### ISSUE TITLE
Where file .ckpt are stored

### ISSUE THREAD

**Issue by FrancescoGigli** [Calamari-OCR/calamari#207]
Hi,
after I train a model with calamari 2.0.0 no files .ckpt were created, but instead a folder train is created.
If I want to use calamari-predict --checkpoint path_to_model.ckpt --files your_images.*.png how i can get the .ckpt files now?

**Comment by ChWick** [comment]
Usually, the ckpt model is places within the `--output_dir`.
I expect that there might be a bug. Did you pass validation data? Alternatively, set `--checkpoint_frequency 1` (or higher) so that a model will be written at the end of each epoch
If there is a `train.log` within the `train` dir, feel free to share it. Maybe this will help to understand this bug.

**Comment by FrancescoGigli** [comment]
Hi, maybe I didn't explain correctly my issue.
the problem is that I have a folder containing varius png and their corresponding gt.txt.
After i call the line  `calamari-train --files *.png` in my folder appears folders _train_ and _checkpoint_.
But for the function `calamari-predict --checkpoint path_to_model.ckpt --files your_images.*.png` I need a file .ckpt that I usually got back in _Calamari 1.5.0_.
My question is: now where I can find this .ckpt files?

**Comment by ChWick** [comment]
This is a bug in calamari 2.0. In the previous Calamari, if no validation data (`--validation`) is provided, the training data is used to determine the best model which is the final model.
Unfortunately, you have to retrain you model using the `--use_train_as_val` flag explicitly in the new Calamari version. Alternatively, you can set `--validation_split_ratio 0.2` to use 20% of your training data for validation (to determine the best model).
After each epoch a model (if it is better than the previous) should be written similar to Calamari 1 in the `--output_dir`

**Comment by FrancescoGigli** [comment]
Ok thank you now I can predict correcly.
If I wanna use a different validation set how i can pass the folder with the file to validate? 
In my case I have a folder named TrainSet with the .png and .gt.txt files to train and a folder named ValidationSet with the .png and .gt.txt for validate. What is the comand to train this 2 folder using the first as train and the second as validation set?
